featuretools/wrappers/sklearn.py

```diff
@@ -0,0 +1,261 @@
+                rows are split per cutoff time.
+
+            dask_kwargs (dict, optional): Dictionary of keyword arguments to be
+                passed when creating the dask client and scheduler. Even if n_jobs
+                is not set, using `dask_kwargs` will enable multiprocessing.
+                Main parameters:
+
+                cluster (str or dask.distributed.LocalCluster):
+                    cluster or address of cluster to send tasks to. If unspecified,
+                    a cluster will be created.
+                diagnostics port (int):
+                    port number to use for web dashboard.  If left unspecified, web
+                    interface will not be enabled.
+
+                Valid keyword arguments for LocalCluster will also be accepted.
+
+            profile (bool, optional): Enables profiling if True.
+
+        Example:
+            .. code-block:: python
+                import featuretools as ft
+                from sklearn.preprocessing import StandardScaler
+
+                # Get examle data
+                es = ft.demo.load_mock_customer(return_entityset=True)
+                df = es['customers'].df
+
+                # Build dataset
+                pipeline = Pipeline(steps=[
+                    ('ft', dfsTransfomer(entityset=es, target_entity="customers", max_features=2)),
+                    ('sc', StandardScaler()),
+                ])
+
+                # Fit and transform
+                pipeline.fit(df['customer_id'].values).transform(df['customer_id'].values)
+
+        """
+        self.entities = entities
+        self.relationships = relationships
+        self.entityset = entityset
+        self.target_entity = target_entity
+        self.agg_primitives = agg_primitives
+        self.trans_primitives = trans_primitives
+        self.allowed_paths = allowed_paths
+        self.max_depth = max_depth
+        self.ignore_entities = ignore_entities
+        self.ignore_variables = ignore_variables
+        self.seed_features = seed_features
+        self.drop_contains = drop_contains
+        self.drop_exact = drop_exact
+        self.where_primitives = where_primitives
+        self.max_features = max_features
+        self.save_progress = save_progress
+        self.training_window = training_window
+        self.approximate = approximate
+        self.chunk_size = chunk_size
+        self.n_jobs = n_jobs
+        self.dask_kwargs = dask_kwargs
+        self.verbose = verbose
+        self.profile = profile
+
+    def fit(self, entity_ids, y=None):
+        """Wrapper for dfs
+
+            Calculates a feature matrix and features given a dictionary of entities and a list of relationships.
+
+            Args:
+                entity_ids (list): List of instances to calculate features on.
+
+            See Also:
+                :func:`synthesis.dfs`
+        """
+        self.feature_defs =    dfs(entities=self.entities,
```

Comment:
because of `features_only`, you don't need the following arguments related to calculation in the fit() method: `save_progress`, `training_window`, `approximate`, `chunk_size`,` n_jobs`, `dask_kwargs`